controller/src/main/java/io/pravega/controller/store/stream/StreamCutRecord.java

```diff
@@ -9,18 +9,25 @@
  */
 package io.pravega.controller.store.stream;
 
+import io.pravega.common.ObjectBuilder;
+import io.pravega.common.io.serialization.VersionedSerializer;
+import io.pravega.controller.store.stream.tables.serializers.StreamCutRecordSerializerV1;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
 import lombok.Data;
 
-import java.io.Serializable;
 import java.util.Map;
 
-
 /**
  * This is data class for storing stream cut with time when the cut was computed.
  * And the size of data being cut.
  */
 @Data
-public class StreamCutRecord implements Serializable {
+@Builder
+@AllArgsConstructor
+public class StreamCutRecord {
+    public static final VersionedSerializer.WithBuilder<StreamCutRecord, StreamCutRecordBuilder> SERIALIZER_V1 = new StreamCutRecordSerializerV1();
```

Comment:
1. In the code, we'll never have `V1` or `V<N>` for serializers. We'll always have a single serializer that knows all the versions it supports. Inside the serializer it declares the versions and for each version, the revisions and their associated implementations. So we can just name this `SERIALIZER`
2. To keep the line shorter, feel free to declare the field of type `StreamCutRecordSerializerV1` (maybe without `V1` if you agree with the point above).